hermes-management/src/main/java/pl/allegro/tech/hermes/management/config/SubscriptionServiceConfiguration.java

```diff
@@ -0,0 +1,54 @@
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import pl.allegro.tech.hermes.domain.subscription.SubscriptionRepository;
+import pl.allegro.tech.hermes.management.domain.Auditor;
+import pl.allegro.tech.hermes.management.domain.dc.MultiDatacenterRepositoryCommandExecutor;
+import pl.allegro.tech.hermes.management.domain.dc.RepositoryManager;
+import pl.allegro.tech.hermes.management.domain.subscription.SubscriptionMetricsRepository;
+import pl.allegro.tech.hermes.management.domain.subscription.SubscriptionOwnerCache;
+import pl.allegro.tech.hermes.management.domain.subscription.SubscriptionService;
+import pl.allegro.tech.hermes.management.domain.subscription.health.SubscriptionHealthChecker;
+import pl.allegro.tech.hermes.management.domain.subscription.validator.SubscriptionValidator;
+import pl.allegro.tech.hermes.management.domain.topic.TopicService;
+import pl.allegro.tech.hermes.management.infrastructure.kafka.MultiDCAwareService;
+import pl.allegro.tech.hermes.tracker.management.LogRepository;
+
+import java.util.concurrent.Executors;
+
+@Configuration
+@EnableConfigurationProperties(SubscriptionHealthCheckProperties.class)
+public class SubscriptionServiceConfiguration {
+
+    @Bean
+    public SubscriptionService subscriptionService(SubscriptionRepository subscriptionRepository,
+                                                   SubscriptionOwnerCache subscriptionOwnerCache,
+                                                   TopicService topicService,
+                                                   SubscriptionMetricsRepository metricsRepository,
+                                                   SubscriptionHealthChecker subscriptionHealthChecker,
+                                                   LogRepository logRepository,
+                                                   SubscriptionValidator subscriptionValidator,
+                                                   Auditor auditor,
+                                                   MultiDatacenterRepositoryCommandExecutor multiDcExecutor,
+                                                   MultiDCAwareService multiDCAwareService,
+                                                   RepositoryManager repositoryManager,
+                                                   SubscriptionHealthCheckProperties healthCheckProperties) {
+        return new SubscriptionService(
+                subscriptionRepository,
+                subscriptionOwnerCache,
+                topicService,
+                metricsRepository,
+                subscriptionHealthChecker,
+                logRepository,
+                subscriptionValidator,
+                auditor,
+                multiDcExecutor,
+                multiDCAwareService,
+                repositoryManager,
+                Executors.newFixedThreadPool(healthCheckProperties.getSubscriptionServiceThreads()),
```

Comment:
As far as I remember, it is preferred in the project to create threads with meaningful names using the `ThreadFactoryBuilder` like [here](https://github.com/allegro/hermes/blob/master/hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/health/HealthCheckScheduler.java#L29).